tabfilter.py

```diff
@@ -16,25 +16,35 @@ class TabFilterCommand(sublime_plugin.WindowCommand):
 
 	def run(self):
 		"""Shows a quick panel to filter and select tabs from the active window"""
-		
+
 		tabs = []
 		self.window = sublime.active_window()
 		self.views = []
 		self.prefix = ""
 		self.settings = sublime.load_settings("tabfilter.sublime-settings")
 
+		idx = 0
 		for view in self.window.views():
 			self.views.append(view)
+			if self.window.active_sheet().view().id() == view.id():
+				""" save index for later usage"""
+				self.current_tab_idx = idx
```

Comment:
Could we define a default for this alongside the other class properties such as `self.prefix` a little higher up in the file?  I'm guessing based on the code later in the event handler, we'd want to default to something like `-1`.  Otherwise, trying to run the plugin with no open tabs results in an `AttributeError`